mailboxes/mailboxes-lib.pl

```diff
@@ -1286,14 +1286,32 @@ sub user_read_dbm_file
 return $rv;
 }
 
-# parse_calendar_file(calendar-file)
+# parse_calendar_file(calendar-file|lines)
 # Parses an iCalendar file and returns a list of events
 sub parse_calendar_file
 {
 my ($calendar_file) = @_;
-my (@events, %event);
+my (@events, %event, $line);
 eval "use DateTime; use DateTime::TimeZone;";
 return \@events if ($@);
+# Timezone map
+my %timezone_map = (
```

Comment:
Do we really need this? Seems like there a lot of timezones that aren't covered

Reply:
Yes, we do need this. This is what ICS is using. Which one aren't covered? Do you have the list?

Reply:
I mean, surely there are more timezones in the world than this

Reply:
Yes, for sure! Though those are just to cover zones used by ICS.

Reply:
So ICS only uses those small number of timezones? Is this documented somewhere?

Reply:
No, it isn't. Fix on the latest commit.

Reply:
To be clear, zones like `Asia/Nicosia` or `America/Los_Angeles` don't need to be listed here, as they work perfectly fine with the `DateTime::TimeZone` module.

Reply:
Are there two lists of timezones in this PR now? I also see a list in parse_calendar_file

Reply:
> Are there two lists of timezones in this PR now? I also see a list in parse_calendar_file

You're right! Fixed now!